### PR TITLE
[IOPAE-298] add pgClient and serviceReviewDao

### PR DESCRIPTION
**Pull request by rodoviti**
add pgClient and serviceReviewDao

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/io-services-cms-webapp/src/config.ts [hunk 1]
```diff
@@ -27,12 +29,27 @@ export const JiraConfig = t.interface({
...
+  DB_USER: NonEmptyString,
+});
+
 // global app configuration
 export type IConfig = t.TypeOf<typeof IConfig>;
 export const IConfig = t.intersection([
   t.type({ isProduction: t.boolean }),
   InternalStorageAccount,
   JiraConfig,
+  IDecodableConfigPostgreSQL,
```
Comment (giuseppedipinto) [inline apps/io-services-cms-webapp/src/config.ts:52]: Should we use the same type naming convention, i.e. without the leading "I"?

Comment (rodoviti) [reply apps/io-services-cms-webapp/src/config.ts:52]: honestly I agree with you. I just copied and pasted that naming from other repo 😄 
@balanza what do you think?

File: apps/io-services-cms-webapp/src/config.ts [hunk 2]
```diff
@@ -27,12 +29,27 @@ export const JiraConfig = t.interface({
 });
 export type JiraConfig = t.TypeOf<typeof JiraConfig>;
 
+export type IDecodableConfigPostgreSQL = t.TypeOf<
+  typeof IDecodableConfigPostgreSQL
+>;
+export const IDecodableConfigPostgreSQL = t.interface({
+  DB_HOST: NonEmptyString,
```
[the remark this answers is not in the document]
Comment (rodoviti) [reply apps/io-services-cms-webapp/src/config.ts]: fixed with 781941c549d7d37440c4eee2ff50127b802aef28

File: apps/io-services-cms-webapp/src/config.ts (hunk 1 above)
Comment (balanza) [reply apps/io-services-cms-webapp/src/config.ts:52]: Fine for me

Comment (rodoviti) [reply apps/io-services-cms-webapp/src/config.ts]: fixed with c5e0eb1ff8a663800955a65399d6d333d8a522e2

File: apps/io-services-cms-webapp/src/config.ts [hunk 3]
```diff
@@ -27,12 +33,26 @@ export const JiraConfig = t.interface({
...
+export const PostgreSqlConfig = t.interface({
+  REVIEWER_DB_HOST: NonEmptyString,
+  REVIEWER_DB_IDLE_TIMEOUT: withDefault(NumberFromString, 30000),
+  REVIEWER_DB_NAME: NonEmptyString,
+  REVIEWER_DB_PASSWORD: NonEmptyString,
+  REVIEWER_DB_PORT: NumberFromString,
+  REVIEWER_DB_SCHEMA: NonEmptyString,
+  REVIEWER_DB_TABLE: NonEmptyString,
+  REVIEWER_DB_USER: NonEmptyString,
+  REVIEWER_DB_READ_MAX_ROW: withDefault(IntegerFromString, 50),
```
Comment (rodoviti) [inline apps/io-services-cms-webapp/src/config.ts:46]: @balanza @giuseppedipinto  Could this be a proper default value?

**Review by giuseppedipinto — APPROVED** [review]

**Pull request merged**